docs/dist/index.html

```diff
@@ -11,7 +11,7 @@
     <script src="/dist/assets/jquery-2.2.4.min.js"></script>
     <link rel="stylesheet" href="//d5wfroyti11sa.cloudfront.net/prod/client/ts-10.0.28.min.css">
     <link rel="stylesheet" href="/dist/assets/dox.css">
-    <link rel="icon" href="data:;base64,iVBORw0KGgo=">
+    <link rel="icon" href="https://developers.tradeshift.com/icons/favicon-32x32.png">
```

Comment:
We should probably move this icon to be part of our codebase, either base64 or just deploy the file.

Maybe we should have our own icon which is similar to it, but has some sort of `ts.ui` feel. What do you think @mihneaz ?